app/components/emoji/index.ts

```diff
@@ -41,11 +37,7 @@ function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
         imageUrl = Client4.getCustomEmojiImageUrl(emoji!.id);
         isCustomEmoji = true;
     } else {
-        displayTextOnly = state.entities.emojis.nonExistentEmoji.has(emojiName) ||
-            config.EnableCustomEmoji !== 'true' ||
-            config.ExperimentalEnablePostMetadata === 'true' ||
-            getCurrentUserId(state) === '' ||
-            isMinimumServerVersion(Client4.getServerVersion(), 5, 12);
+        displayTextOnly = true;
```

Comment:
why set to `true` directly without checking the other ones?

Reply:
This is an OR chain. The moment one of the values is true, it is always true. Since we are always after server version 5,12, this is always true.